web/ui/static/css/graph.css

```diff
@@ -6,6 +6,21 @@ body {
   font-size: 11px;
 }
 
+div.query-history {
+  margin-top: 20px;
+  margin-left: 15px;
+}
+
+div.query-history button {
+  font-size: 0.8em;
+}
+
+div.query-history button.is-checked {
```

Comment:
In #3445 I add a similar button, and in whichever PR goes in last I can refactor and reuse the checkbox button code.